compiler/luci-interpreter/src/kernels/Utils.h

```diff
@@ -31,6 +31,11 @@ namespace luci_interpreter
 namespace kernels
 {
 
+#define THROW_IF_CONDITION_FALSE(cond)                                                \
+  if (!(cond))                                                                        \
+    throw std::runtime_error(std::string(__FILE__) + ":" + std::to_string(__LINE__) + \
```

Comment:
It may be worth to also include the `cond` in the output.

Reply:
Okay, I will fix it.